docs/api/context-bridge.md

```diff
@@ -46,11 +46,17 @@ The `contextBridge` module has the following methods:
 * `apiKey` string - The key to inject the API onto `window` with.  The API will be accessible on `window[apiKey]`.
 * `api` any - Your API, more information on what this API can be and how it works is available below.
 
+### `contextBridge.exposeInIsolatedWorld(worldId, apiKey, api)`
+
+* `worldId` Integer - The ID of the world to inject the API into. This has to be an existing world.
```

Comment:
iirc `999` is the world ID that the preload uses. What happens if you try to expose a method in world `999`? Should we do anything to prevent that? Should we at least note something about that?

Reply:
Exposing functionality in preload context could be a reasonable expectation, but it's not the concern for the functionality of exposeInIsolatedWorld.

I will mention using 999 would expose the object for preload context in the documentation :man-bowing 🙇‍♂️

Reply:
Please also include a test which calls `exposeInIsolatedWorld` with that world ID.